src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss

```diff
@@ -288,14 +289,15 @@
   .user-search {
     height: 100%;
     width: 100%;
-    position: relative;
+    position: absolute;
     z-index: 9;
     top: 0;
     left: 0;
     background-color: rgba(0, 0, 0, 0.6);
     .list-container {
       top: 12em;
       width: 80%;
+      position: relative;
```

Comment:
dont overwrite the position to relative and use transform to make it in center of div
we have mixin @include transform(translateX(-50%)); left:50%